python/prophet/plot.py

```diff
@@ -541,8 +541,8 @@ def plot_cross_validation_metric(
         if np.timedelta64(1, dt) < np.timedelta64(tick_w, 'ns'):
             break
 
-    x_plt = df_none['horizon'].astype('timedelta64[ns]').astype(np.int64) / float(dt_conversions[i])
-    x_plt_h = df_h['horizon'].astype('timedelta64[ns]').astype(np.int64) / float(dt_conversions[i])
+    x_plt = df_none['horizon'].view('timedelta64[ns]').view(np.int64) / float(dt_conversions[i])
```

Comment:
I think the warning is referring specifically to the second cast, i.e. `.astype(np.int64)`. The first cast to a timedelta shouldn't change.

Reply:
Got it - should have just made that edit.